lib/components/table/table.test.ts

```diff
@@ -0,0 +1,363 @@
+                data-testid="test-sortable-${modifier}-svg-${icon.direction}"
+                aria-hidden="true"
+                class="svg-icon js-sorting-indicator ${icon.classes}"
+                width="14"
+                height="14"
+                viewBox="0 0 14 14"
+            >
+                <path d="${icon.pathD}"></path>
+            </svg>
+        `
+    );
+
+const getTh = (label: string, modifier: string, useButton: boolean) =>
+    useButton
+        ? html`
+              <th
+                  data-s-table-target="column"
+                  data-testid="test-sortable-${modifier}-col"
+              >
+                  <button
+                      data-action="click->s-table#sort"
+                      data-testid="test-sortable-${modifier}-btn"
+                  >
+                      ${label} ${sortDirectionIcons(modifier)}
+                  </button>
+              </th>
+          `
+        : html`
+              <th
+                  data-s-table-target="column"
+                  data-action="click->s-table#sort"
+                  data-testid="test-sortable-${modifier}-col"
+              >
+                  ${label} ${sortDirectionIcons(modifier)}
+              </th>
+          `;
+
+const mockSortableTable = (useButton: boolean) => {
+    return html`
+        <div class="s-table-container" data-testid="test-sortable-table">
+            <table class="s-table s-table__sortable" data-controller="s-table">
+                <thead>
+                    <tr>
+                        ${getTh("Season", "season", useButton)}
+                        ${getTh("Month", "month", useButton)}
+                        ${getTh("Temperature", "temp", useButton)}
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>Winter</td>
+                        <td data-s-table-sort-val="12">December</td>
+                        <td>2</td>
+                    </tr>
+                    <tr>
+                        <td>Spring</td>
+                        <td data-s-table-sort-val="3">March</td>
+                        <td>13</td>
+                    </tr>
+                    <tr>
+                        <td>Summer</td>
+                        <td data-s-table-sort-val="6">June</td>
+                        <td>25</td>
+                    </tr>
+                    <tr>
+                        <td>Fall</td>
+                        <td data-s-table-sort-val="9">September</td>
+                        <td>13</td>
+                    </tr>
+                    <tr data-s-table-sort-to="bottom" class="fw-bold">
+                        <td colspan="2">Average temperature</td>
+                        <td>13</td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    `;
+};
+
+describe("s-table", () => {
+    [true, false].forEach((useButton) => {
+        describe("Column sorting indicators", () => {
```

Comment:
```suggestion
    [true, false].forEach((useButton) => {
        describe(`Column sorting indicators (${useButton ? '' : 'not '}using legacy markup)`, () => {
```

Can you do something like this so that we can read the output of the test results a bit better about when they/if fail

Reply:
Absolutely! Funny thing is I had something similar in place junked it while trying to debug an issue.